Euroleague-Graduate/Euroleague-Graduate/TurkishAirLinesHelper.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  TurkishAirLinesHelper.swift
+//  Euroleague-Graduate
+//
+//  Created by Modeso on 7/19/17.
+//  Copyright © 2017 Modeso. All rights reserved.
+//
+
+import Foundation
+
```

Comment:
This is a layer that uses the Api Client.. 
so it's better to have the layers like the following:
ApiClient -> Have the get, post methods
TurkishAirlinesApiClient, or GamesApiClient
so that if we needed to add games or other leagues (competitions) we don't want to repeat the code